## Hi there ðŸ‘‹

My name is Rachel and in this repository you will find all supplementary data files cited in my dissertation, "The Physical and Molecular Responses of Pennycress (Thlaspi arvense L.) to Waterlogging", and in my manuscript "Morpho-physiological and transcriptomic responses of field pennycress to waterlogging".

If you would like access to my code for my RNA-seq analyses, please see my "Code-for-RNA-seq_Analysis" Repository.


The following is a brief description of the supplementary data files found in this repository:

-Supplemental_File_1_DEGs.xlsx : This file contains a list of all the differentially expressed genes in root, leaf, and pod tissues based on 4 conditions (control, 7d waterlogged, 3d waterlogged, and 3HR recovery) and 2 accessions (MN106 and SP32-10).

-Supplemental_File_2_GSEA-GO.xlsx : This file contains the Gene Ontology (GO) terms of the clusterProfiler output from the Gene Set Enrichment Analysis of each analysis.

-Supplemental_File_3_GO-genes.xlsx : This file contains a list of the most significant DEGs and their Arabidopsis ortholog, log2FC, and padj. They are grouped by some of the most significant GO categories.

-Supplemental_File_4_GSEA-KEGG.xlsx : This file contains the KEGG terms of the clusterProfiler output from the Gene Set Enrichment Analysis of each analysis.

-Supplemental_File_5_139-DEGs.xlsx : This file contains a list of the 139 DEGs in the interaction of MN106 7d waterlogged vs control roots compared to SP32-10 7d waterlogged vs control roots (MN-7WL vs SP-7WL). Arabidopsis gene IDs and names are included.

-Supplemental_File_6_46-DEGs.xlsx : This file contains a list of the 46 DEGs in the interaction of SP32-10 7d waterlogged vs control roots compared to SP32-10 3d waterlogged vs control roots (SP-7WL vs SP-3WL). Arabidopsis gene IDs and names are included.
